include/boost/beast/core/detail/is_invocable.hpp

```diff
@@ -12,6 +12,8 @@
 
 #include <type_traits>
 #include <utility>
+#include <boost/asio/compose.hpp>
```

Comment:
Use `<boost/asio/async_result.hpp>` instead?

Reply:
good catch. I started with async_compose and then realised I could get away with less.